packages/ua-devtools/src/oapp/config.ts

```diff
@@ -75,10 +73,41 @@ export const configureOAppPeers: OAppConfigurator = async (graph, createSdk) =>
 
                 logger.verbose(`Creating a connection ${formatOmniVector({ from, to })}`)
                 return [await sdk.setPeer(to.eid, to.address)]
-            })
+            },
+            {
+                onStart: (
+                    logger,
+                    [
+                        {
+                            vector: { from, to },
```

Comment:
Here you don't need to destructure and restructure - just do:

```typescript
onStart: (logger, [{ vector }]) => logger.verbose(`Checking connection ${formatOmniVector(vector)}`),
```

Reply:
Makes sense !